app/Http/Middleware/isUserMiddleware.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class isUserMiddleware
```

Comment:
bu middleware i şuan için deactive edelim. Kalsın ama kullanılmasın. ihtiyaç durumna göre açarız. Kernel.php den de kaldır yani